src/Bridge/ApiPlatform/Core/JsonSchema/Type/ElaoEnumType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We're not using strict types yet in the code base. Let's remove it for now.

Reply:
Removed.